src/computer.h

```diff
@@ -116,6 +116,8 @@ class computer
     private:
         // Difficulty of simply logging in
         int security;
+        // Date of next attempt
+        long next_attempt = 0;
```

Comment:
I'm a bit worry about size of ```int``` (I'm not sure if its size is enough for long-run game), that's why I used ```long```. But as you correctly state, it's probably useless here, so I need to go back to ```int```.